spec/haskell/src/SEL4/Object/Notification.lhs

```diff
@@ -72,14 +72,15 @@ mark the notification object as active.
 >                         schedulable <- getSchedulable tcb
 >                         when schedulable $ possibleSwitchTo tcb
 >                         scOpt <- threadGet tcbSchedContext tcb
->                         ifCondRefillUnblockCheck scOpt (Just True) (Just True)
+>                         ifCondRefillUnblockCheck scOpt (Just False) (Just False)
```

Comment:
Thanks for the commit message on that, I was temporarily quite confused how we can just change this.